Goutte/Client.php

```diff
@@ -157,9 +170,9 @@ protected function addPostFiles(array $files, array &$multipart, $arrayName = ''
                 $name = $arrayName.'['.$name.']';
             }
 
-            $file = [
+            $file = array(
```

Comment:
@stof is right, these changes should be reverted. Sorry about the confusion, fabbot is quite dumb sometimes :)